crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -371,6 +371,7 @@ impl<'w, 's, 'a> EntityCommands<'w, 's, 'a> {
     /// }
     /// # my_system.system();
     /// ```
+    #[must_use = "Omit the .id() call if you do not need to store the `Entity` identifier."]
     #[inline]
```

Comment:
I think we should move `#[inline]` above `#[must_use]` or do it the other way around on the others to keep it consistent.